src/migratus/cli.clj

```diff
@@ -119,13 +130,63 @@
          logger (.getLoggerName record)]
      (core/format fmt date src logger level msg thr))))
 
+(defn verbose-log-level [v]
```

Comment:
So if I don't specify any verbosity level, it will be default = 0 . 
That means that only SEVERE messages (error / fata) will be logged.

I wonder if that is ok. 
The messages with INFO level are very useful IMO.

Reply:
Yeah, I think defaulting to info level would be better.

Reply:
INFO includes the above levels: error and warn. 
Later it might be good to investigate using multiple loggers and ConsoleHandlers to direct correct output to stdout and errors to stderr to play nice with shell redirection.

Reply:
Yeah, I think that'll be fine for the first pass, splitting up the logs to correct outputs is a bonus.

Reply:
I have added log level INFO as default.